tests/utils/test_imageio.py

```diff
@@ -1,5 +1,42 @@
 import pytest
 from icevision.all import *
+from icevision.utils.imageio import open_gray_scale_image
+
+
+@pytest.mark.parametrize(
+    "fn,expected",
+    [
+        ("images/test_3_bands_int8.tif", (100, 101, 3)),
+        ("images/test_3_bands_int16.tif", (100, 101, 3)),
+        ("images/test_3_bands_int32.tif", (100, 101, 3)),
+        ("images/test_3_bands_float32.tif", (100, 101, 3)),
+    ],
+)
+def test_open_img_returns_tiff_with_shape_h_w_c_when_when_tiff_has_3_bands_with_gray_is_false(
```

Comment:
Acceptable but I'd opt for describing the test case in docstring, to me its more readable than verbose test func name (due to the possibility  of using all signs and not limited to _ as space). 
Still, this is debatable so I approve

Reply:
While I agree that docstring is much more readable, I'm not sure pytest would display the docstring by default.  
From my understanding it shows the function name and so to get a clear message of what test passed or failed I think the function name is better by default.  
That being said, it looks like it's possible to make pytest show the docstring (see https://stackoverflow.com/a/39035226/7270039 )  
  
Definitely something to look for in the future as I agree docstring would be more readable.  
  
Cheers